docs/data/capture.md

```diff
@@ -34,11 +34,11 @@ However, if you want to manage your machine's captured data yourself, you can en
 
 To capture data from one or more machines, you must first add the [data management service](../):
 
-1. From your machine's **Config** tab, navigate to the **Services** subtab.
-2. Click **Create service** in the lower-left corner of the page.
-   Choose `Data Management` as the type and specify a name for your data management service, for example `data-manager`.
-3. Click **Create**.
-4. On the panel that appears, you can manage the capturing and syncing functions and specify the **directory**, the sync **interval** and any **tags** to apply to captured data.
+1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
```

Comment:
Nit: we tend to use `on your | the page` over `of`:

```suggestion
1. Navigate to the **CONFIGURE** tab on your machine's page in [the Viam app](https://app.viam.com).
```

Reply:
I like of... sorry 😆

Reply:
I think on sounds good too but I don't think it absolutely needs to be changed